src/pages/proposal/create/ExcludeMemberForm.tsx

```
@@ -115,38 +115,41 @@ const ExcludeMemberForm = () => {
-                  consortium, other members of the consortium may vote on your
-                  proposal.
+                  Once you submit a voting proposal to exclude a consortium
+                  member, other members may vote on your proposal.
                 </Typography>
                 <Typography variant="body2" color="text.secondary">
-                  If more than 50% of all consortium members vote
-                  &quot;yes&quot;, the member you stated will be excluded from
-                  the consortium. He will therefore lose his voting privileges,
-                  will be removed from the validators list and will no longer be
-                  able to earn validator rewards, but will keep his already
+                  If more than 50% of all consortium members vote "yes," the
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)